### PR TITLE
MTDSA-35533 - Remove unused resolver files from the Business Details API as part of API decoupling

### PR DESCRIPTION
**Pull request by bkarimii**
The first two commits cover the story changes and the test coverage increase required to meet the minimum threshold. 
The third commit contains the import optimisations.

### CODE REVIEW

File: app/v2/listAllBusinesses/model/response/ListAllBusinessesResponse.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 package v2.listAllBusinesses.model.response
 
-import cats.Functor
+//import cats.Functor
```
Comment (EdCope) [inline app/v2/listAllBusinesses/model/response/ListAllBusinessesResponse.scala]: Please remove this commented import

**Review by EdCope — CHANGES_REQUESTED** [review]

**Review by EdCope — APPROVED** [review]

**Review by EdCope — APPROVED** [review]
